recipes/rb-jekyll-theme-architect/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set theme_name = "architect" %}
+{% set name = '-'.join(['jekyll-theme', theme_name]) %}
+{% set version = "0.1.1" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: cab35b25725cdb90b7f2a1a545688ab647379cb95888c43f49ebf4f030059e6c
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-jekyll >=3.5,<4.0
+    - rb-jekyll-seo-tag >=2.0,<3
+
+test:
+  commands:
+    - gem list -i {{ name }} -v {{ version }}
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: CC0-1.0
+  license_file: {{ name }}-{{ version }}/LICENSE
+  summary: Architect is a Jekyll theme for GitHub Pages
+
+  doc_url: https://www.rubydoc.info/gems/{{ name }}
+  dev_url: https://github.com/pages-theme/{{ theme_name }}
```

This needs to be done for all recipes
```suggestion
  dev_url: https://github.com/pages-themes/{{ theme_name }}
```